assets/javascripts/discourse/connectors/topic-map/topic-map-activity-pub.hbs

```diff
@@ -0,0 +1,6 @@
+{{yield}}
+{{#if this.showAcivityPubTopicMap}}
```

Comment:
```suggestion
{{yield}}

{{#if this.showActivityPubTopicMap}}
```

You're probably quite tired of typing `Activity` :-) 

Also, I wouldn't merge this suggestion as is, but rather, move this component to `.gjs`.

Reply:
> You're probably quite tired of typing Activity :-)

You're not wrong 😆

Reply:
Done.